tensorboard/webapp/feature_flag/views/feature_flag_module.ts

```diff
@@ -23,7 +24,7 @@ import {FeatureFlagPageContainer} from './feature_flag_page_container';
  */
 @NgModule({
   declarations: [FeatureFlagPageComponent, FeatureFlagPageContainer],
-  imports: [CommonModule, MatCheckboxModule],
+  imports: [CommonModule, MatButtonModule, MatSelectModule],
```

Comment:
You should chat with James about this change. It will require getting the internal version of this source allowlisted to use these modules. I believe James had a TODO to get blanket allowlisted for all these Angular component-related modules. Now (ie before this PR is submitted) could be a good time to get that process started.

Reply:
I do remember this. I never followed up. I will submit that request tomorrow after syncing it with the latest.
(for googlers see cl/456268297).

Reply:
Trying to sync that CL was a pain because the allowlists were moved. (New cl for googlers cl/468741117)

Reply:
The Angular team suggests we update to use the new "MDC materials". I am inclined to agree with them. I suggest we keep this as is and Riley add this directory to the allow list. Then we discuss a large scale refactor  to all materials to use the new system. Once we are using the new system we no longer need to update allow lists.

Reply:
The angular team allowed us to change the legacy allowlist to allow the entire webapp directory. Therefore you no longer need to add this directory explicitly to the allowlist.